osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -319,6 +319,18 @@ public void Delete(Expression<Func<BeatmapSetInfo, bool>>? filter = null, bool s
             });
         }
 
+        public void DeleteVideos(Expression<Func<BeatmapSetInfo, bool>>? filter = null, bool silent = false)
```

Comment:
What's the point of the `filter` and `silent` parameters if they are never actually used?

Reply:
Included those as the function that this was adapted from, `Delete()` (for deleting entire beatmaps) also has them. They may be useful in the future, but I can remove the parameters if you insist.

Reply:
Let's stick to what is useful in the here and now, please. Don't plan in advance for things that will likely never be needed.